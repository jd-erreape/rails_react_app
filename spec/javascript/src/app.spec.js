import React from 'react';
import { shallow } from 'enzyme';
import { Navbar } from 'react-bootstrap';
import App from 'src/app';  

describe('App component', () => {
  describe('contains Navbar Brand', () => {
    it('matches snapshot', () => {
      const wrapper = shallow(<App />);

      expect(wrapper).toMatchSnapshot();
    });

    it('contains the proper title', () => {
      const wrapper = shallow(<App />);

      expect(wrapper.find(Navbar.Brand).text()).toBe('Simple Rails / React App')
    });
  });
});