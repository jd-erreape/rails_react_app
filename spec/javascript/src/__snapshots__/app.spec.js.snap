// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component contains Navbar Brand matches snapshot 1`] = `ShallowWrapper {}`;
